packages/dev/inspector-v2/src/services/panes/properties/materialPropertiesService.tsx

```diff
@@ -44,10 +57,46 @@ export const MaterialPropertiesServiceDefinition: ServiceDefinition<[], [IProper
             ],
         });
 
+        const pbrBaseMaterialPropertiesRegistration = propertiesService.addSectionContent({
+            key: "PBR Base Material Properties",
+            predicate: (entity: unknown): entity is PBRBaseMaterial => entity instanceof PBRBaseMaterial && !(entity instanceof PBRBaseSimpleMaterial),

```

Comment:
Can you add a comment on why we don't want this to be applied to `PBRBaseSimpleMaterial`?

Reply:
Chatted with Gary, I am removing the filter.